accounting/bank/setup/manage_cash_register.rst

```diff
@@ -0,0 +1,62 @@
+==============================
+How to manage a cash register?
+==============================
+
+The cash register is a journal to register receivings and payments transactions.
+It calculates the total money in and out, computing the total balance.
+
+Configuration
+=============
+
+.. image:: ./media/journal.png
+   :align: center
+
+1. Configure the Cash journal in :menuselection:`Accounting --> Configuration --> 
+   Journals`.
+
+2. In the tab Journal Entries, the Default Debit and Credit Account can be
+   configured as well as the currency of the journal
+
+
+.. demo:fields:: account.action_account_journal_form
+
+
+Usage
+=====
+
+How to register cash payments?
+------------------------------
+
+To register a cash payment specific to another customer, you should follow
+these steps:
+
+1. Go to :menuselection:`Accounting --> Dashboard --> Cash --> Register
+   Transactions`
+
+2. Fill in the start and ending balance
+
+3. Register the transactions, specifying the customers linked to the transaction
+
+Put money in
+------------
+
+Put money in is used to placed your cash manually before starting your
+transactions. From the Register Transaction windows, go to :menuselection:`More
```

Comment:
It's the `Accounting --> Dashboard --> Cash --> Register Transactions` above. Not clear ?